knowledge-base/splitter-save-and-load-state.md

```diff
@@ -95,7 +95,7 @@ The example below demonstrates the described approach.
     }
 }
 ````
-````Service
+````C# Service
```

Comment:
Should we use `C#` or `CS` ? If I am not mistaken, `cs` is the commonly used language name.

Reply:
I think we (with @radkostanev and @xristianstefanov) discussed that the `C#` is actually the more common one. I am not able to find some statistics that shows the opposite, though. Have you come across any?